src/types.rs

```diff
@@ -260,17 +260,28 @@ where
     }
 }
 
-impl ToString for HWIDeviceType {
-    fn to_string(&self) -> String {
+struct Point {
+    x: usize,
+    y: usize,
+}
+
+impl std::fmt::Display for Point {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "({}, {})", self.x, self.y)
+    }
+}
```

Comment:
Did you mean to commit this in?

Reply:
No, might've slipped when using the IDE to fix the clippy, I'll fix it.